spynnaker/pyNN/utilities/utility_calls.py

```diff
@@ -275,18 +282,27 @@ def low(dist):
     return stats.low(dist)
 
 
-def validate_mars_kiss_64_seed(seed):
+def _validate_mars_kiss_64_seed(seed):
     """ Update the seed to make it compatible with the RNG algorithm
     """
     if seed[1] == 0:
         # y (<- seed[1]) can't be zero so set to arbitrary non-zero if so
-        seed[1] = 13031301
+        seed[1] = ARBITRARY_Y
 
     # avoid z=c=0 and make < 698769069
-    seed[3] = seed[3] % 698769068 + 1
+    seed[3] = seed[3] % MARS_C_MAX + 1
     return seed
 
 
+def create_mars_kiss_seeds(rng, seed):
```

Comment:
Needs a little bit of a doc comment here.

Reply:
oops. fixing

Reply:
doced